partner-chains-cli/src/prepare_configuration/prepare_main_chain_config.rs

```diff
@@ -126,6 +129,31 @@ fn run_pc_contracts_cli_addresses<C: IOContext>(
 			.ok_or(anyhow!("Permissioned candidates policy id from pc-contracts-cli addresses command output cannot be converted to string"))?
 			.to_string(),
 		context);
+	ILLIQUID_SUPPLY_ADDRESS.save_to_file(
+		&addresses_json.pointer("/addresses/IlliquidCirculationSupplyValidator")
+			.ok_or(anyhow!("Illiquid circulation supply validator address is missing from pc-contracts-cli addresses command output"))?
+			.as_str()
+			.ok_or(anyhow!("Illiquid circulation supply validator address from pc-contracts-cli addresses command output cannot be converted to string"))?
+			.to_string(),
+		context,
+	);
+	Ok(())
+}
+
+fn prepare_native_token<C: IOContext>(context: &C) -> anyhow::Result<()> {
+	context.print(
+		"Partner Chains can store their initial token supply on Cardano as Cardano native tokens.",
+	);
+	context.print("Creation of the native token is not supported by this wizard and must be performed manually before this step.");
+	if context.prompt_yes_no("Do you want to configure a native token for you Partner Chain?", true)
+	{
+		NATIVE_TOKEN_POLICY.prompt_with_default_from_file_and_save(context);
+		NATIVE_TOKEN_ASSET_NAME.prompt_with_default_from_file_and_save(context);
+	} else {
+		NATIVE_TOKEN_POLICY.save_to_file(&PolicyId::default().to_hex_string(), context);
+		NATIVE_TOKEN_ASSET_NAME.save_to_file(&"0x0".into(), context);
```

Comment:
Isn't it a bug? `0x0` doesn't seem to be a syntactically correct hex string. `0x` should be empty one, right?

Reply:
`0x0` is a valid hex string, I tested it.

`0x` is also valid, but to me it looks invalid so I went with `0x0`.

Reply:
So, `0x` is empty hex - reasonable default.
`0x0` is parsed to one byte with value `0`. I still think that `0x` is better choice.